src/kOS/Suffixed/StageValues.cs

```diff
@@ -1,28 +1,54 @@
 ﻿using System;
+using System.Linq;
 using kOS.Safe.Encapsulation;
-using kOS.Utilities;
+using kOS.Safe.Encapsulation.Suffixes;
 
 namespace kOS.Suffixed
 {
     public class StageValues : Structure
     {
-        private readonly Vessel vessel;
+        private readonly SharedObjects shared;
 
-        public StageValues(Vessel vessel)
+        public StageValues(SharedObjects shared)
         {
-            this.vessel = vessel;
+            this.shared = shared;
+
+            InitializeSuffixes();
+        }
+
+        private void InitializeSuffixes()
+        {
+            AddSuffix("RESOURCES", new Suffix<SuffixedList<ActiveResourceValue>>(GetResourceManifest));
```

Comment:
Hmm.... It occurs to me reading this that our new development standard that says all changes must be accompanied by user docs to do with them doesn't really account for the case of a feature that is being *deliberately* left undocumented for now but is intended to be exposed later on.

I agree about making STAGE:RESOURCES and SHIP:RESOURCES congruent.